### DISCUSSION TITLE
Does this support multiple institutions?

### DISCUSSION THREAD

**Discussion by phirestalker** [ebridges/plaid2qif#37] [Q&A]
Reading the readme file makes it sound like it only works with one institution. Is this capable of working with multiple institutions as long as they are linked with plaid? If not, would I be able to set up different config files for each and feed it to the program?

**Comment by bhipple** [comment]
Yes!  I use it with ~5-10 institutions without issue.  Each institution's credentials go into a separate file and you pass that file via `ACCESS_TOKEN_FILE` on each run.
